src/bacnet/basic/npdu/h_npdu.c

```diff
@@ -194,6 +195,15 @@ static void network_control_handler(BACNET_ADDRESS *src,
         default:
             break;
     }
+/* FIXME GG The following doesn't compile for the microcontroller ports. An appropriate ifdef is needed.
+Using MSTP so far */
+#ifndef BACDL_MSTP
+    if (npdu_data->network_message_type >= 0x14 && npdu_data->network_message_type < 0x80)
```

Comment:
The magic numbers 0x14 and 0x80 are easier to maintain if they exist in the bacenum.h file in the BACNET_NETWORK_MESSAGE_TYPE enumeration:
```
    NETWORK_MESSAGE_ASHRAE_RESERVED_MIN = 20,
    NETWORK_MESSAGE_ASHRAE_RESERVED_MAX = 127,
```

Reply:
Fixed, thanks